### PR TITLE
Add UUIDFilter to filters.__all__

### PR DESCRIPTION
**Pull request by unklphil**
It's currently not possible to import UUIDFilter, because it's not included in `filters.__all__`.

I've added it and alphabetised the list to make it easier to pick these problems up in the future.

### CODE REVIEW

File: django_filters/filters.py [hunk 1]
```diff
@@ -20,13 +20,28 @@
...
-    'DurationFilter',
+    'AllValuesFilter',
+    'BooleanFilter',
+    'CharFilter',
+    'ChoiceFilter',
+    'DateFilter',
+    'DateFromToRangeFilter',
+    'DateRangeFilter',
+    'DateTimeFilter',
+    'DateTimeFromToRangeFilter',
```
Comment (rpkilby) [inline django_filters/filters.py]: looks like you missed `DurationFilter`

Comment (unklphil) [inline django_filters/filters.py]: Thanks, it looks like it was another one that wasn't included in `__all__` to begin with.

**Comment by carltongibson** [comment]
@unklphil This is good. But I think I can see `UUIDFilter` in the list before the change. Can you just confirm what's going on here? (I'm a bit confused)

**Comment by rpkilby** [comment]
I think @unklphil created this based on the 0.13 release, not realizing that the UUIDFilter import was fixed in #389 (which hasn't been released yet).

**Comment by carltongibson** [comment]
@rpkilby Perfect. Thanks. 

Let's have this, it's cleaner. 

@unklphil Thanks for the PR!

**Comment by unklphil** [comment]
Yeah, I didn't realise there was another MR for this issue.

**Comment by unklphil** [comment]
cool, thanks :)

**Pull request merged**